### DISCUSSION TITLE
Chat command that give Random % Number

### DISCUSSION THREAD

**Discussion by KueSupreme** [Sehelitar/Kick.bot#14] [Q&A] [answered]
Hello, so I'm trying to set up user cooldowns, when active none of my commands at all, work global cooldowns work fine but UCDs don't need help

**Comment by Sehelitar** [comment]
You are right, there is a bug with users cooldowns. I opened an issue for this #15 
I'll look into it.

**Comment by Sehelitar** [comment, marked as answer]
> Hello, so I'm trying to set up user cooldowns, when active none of my commands at all, work global cooldowns work fine but UCDs don't need help

A new release is now available with a fix for this problem:
https://github.com/Sehelitar/Kick.bot/releases/tag/v0.3.6.1

**Reply by Chico3tm** [reply]
@Sehelitar, amazing moving up on my favorite ppl list

**Comment by Sehelitar** [comment]
Hey @KueSupreme 👋
Does the update fix your problem?

**Reply by Chico3tm** [reply]
it works for me. the only thing I'm seeing that still has an issue is follow and host alerts working here and there. But I'm starting to think that is a kick problem. Like I didn't any alert triggers while streaming from streamer.bot but my lumia light effects still triggered. But the moment I wasn't streaming the alerts triggered when I got a follow in lumia, streamer.bot and obs.
